recipes/cad1/task2/baseline/car_scene_acoustics.py

```diff
@@ -0,0 +1,260 @@
+"""A class for the car acoustic environment."""
+# pylint: disable=import-error
```

Comment:
```
recipes/cad1/task2/baseline/baseline_utils.py:9:0: E0401: Unable to import 'librosa' (import-error)
recipes/cad1/task2/baseline/baseline_utils.py:11:0: E0401: Unable to import 'pandas' (import-error)
recipes/cad1/task2/baseline/baseline_utils.py:12:0: E0401: Unable to import 'omegaconf' (import-error)
```

Reply:
How would you fix these?

Reply:
Tricky as I'm assuming you already have them installed in your virtual environment and these are false positives.

Are you using VSCode to run `pylint`? If so [this might help solve the problem](https://stackoverflow.com/questions/51095449/pylint-false-positive-e0401-import-errors-in-vscode-while-using-venv).

If its cropping up in the `pre-commit` checks I'll have to think/do more digging.

Reply:
No, I use PyCharm as my IDE.
And yes, I have these errors when I try to commit the changes.
I even have this error when running the pylint from the terminal.

![image](https://user-images.githubusercontent.com/17277802/224481852-f48178af-4f99-4914-b621-b73144086488.png)

Reply:
Btw, this is not the only file where I have this issue.
I had the same problem when I was working on Task1

Reply:
Is this a viable solution for us?
https://stackoverflow.com/a/39207275